### PR TITLE
Added four reactions to Lai_Hexylbenzene Library

### PR DESCRIPTION
**Pull request by laitcl**

### CODE REVIEW

**Comment by mliu49** [comment]
Could you rebase and add a blank line to the end of the reactions.py file?

**Comment by laitcl** [comment]
I actually have new calculations coming in. Will fix while I add the new numbers around next week

**Comment by alongd** [comment]
@laitcl, seems like this branch isn't properly rebased (contains many commits already present in the repo). Could you check it out?

**Comment by laitcl** [comment]
@alongd MINE MINE MINE

File: input/thermo/libraries/Lai_Hexylbenzene.py [hunk 1]
```diff
@@ -840,19 +840,19 @@
     thermo = NASA(
         polynomials = [
             NASAPolynomial(
-                coeffs = [4.05393, -0.00590968, 0.000256082, -4.22713e-07, 2.21322e-10, 4179.59, 11.8847],
```
Comment (alongd) [inline input/thermo/libraries/Lai_Hexylbenzene.py]: The changes in the thermo Lai_Hexylbenzene library look technically OK. Could you add a comment to the commit message explaining the reason for the different values?

File: input/kinetics/libraries/Lai_Hexylbenzene/reactions.py [hunk 2]
```diff
@@ -49,10 +56,7 @@
     shortDesc = u"Calculation performed by Lawrence Lai, December 2017, CBS-QB3 level of theory",
     longDesc = 
 """
-Calculation for Hexylbenzene + H in the substituted position to form CCCCCCC1C=C[CH]C=C1
-Level of Theory: CBS-QB3
-Number of Rotors included: 6
```
Comment (alongd) [inline input/kinetics/libraries/Lai_Hexylbenzene/reactions.py]: Do you think specifying the number of rotors included ins't important/relevant any more?

File: input/thermo/libraries/Lai_Hexylbenzene.py [hunk 3]
```diff
@@ -4458,18 +4458,18 @@
 4  C u0 p0 c0 {3,D} {5,S} {10,S}
 5  C u0 p0 c0 {4,S} {6,D} {15,S}
 6  C u0 p0 c0 {5,D} {7,S} {16,S}
-7  C u0 p0 c0 {6,S} {8,S} {17,S} {18,S}
```
Comment (alongd) [inline input/thermo/libraries/Lai_Hexylbenzene.py]: I think these changes belong to the thermo commit

Comment (laitcl) [reply input/thermo/libraries/Lai_Hexylbenzene.py]: I've regrouped my commits to properly reflect this.

File: input/thermo/libraries/Lai_Hexylbenzene.py (hunk 1 above)
Comment (laitcl) [reply input/thermo/libraries/Lai_Hexylbenzene.py]: Changed commit message to be much more detailed to highlight the changes that took place.

**Comment by alongd** [comment]
@cgrambow  and @mliu49,
We get travis errors for the ML module, such as:
```======================================================================
ERROR: Test that we can make a prediction using MLEstimator.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/ml/estimator_test.py", line 53, in setUp
    self.ml_estimator = MLEstimator(Hf298_path, S298_path, Cp_path)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/ml/estimator.py", line 60, in __init__
    self.hf298_estimator, self.hf298_uncertainty = load_estimator(hf298_path)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/ml/estimator.py", line 129, in load_estimator
    estimator.load_input(input_file)
  File "/home/travis/miniconda/envs/rmg_env/lib/python2.7/site-packages/dde/predictor.py", line 66, in load_input
    read_input_file(self.input_file, self)
  File "/home/travis/miniconda/envs/rmg_env/lib/python2.7/site-packages/dde/input.py", line 72, in read_input_file
    raise e
IOError: [Errno 2] No such file or directory: '/home/travis/build/ReactionMechanismGenerator/RMG-database/input/thermo/ml/main/H298/predictor_input.py'
======================================================================
ERROR: Test that we can input.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/rmg/inputTest.py", line 109, in testMLEstimator
    inp.mlEstimator(thermo=True)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/rmg/input.py", line 395, in mlEstimator
    raise InputError('Cannot find ML models folder {}'.format(models_path))
InputError: Cannot find ML models folder /home/travis/build/ReactionMechanismGenerator/RMG-database/input/thermo/ml/main
```

Could you take a look?
Thanks!

**Comment by cgrambow** [comment]
Rebase this branch. That will fix it.

**Comment by alongd** [comment]
Looking good. Is it ready to be merged?

**Comment by laitcl** [comment]
Okay to go!

> Looking good. Is it ready to be merged?

**Pull request merged**